src/content/docs/apm/agents/php-agent/configuration/distributed-tracing-php-agent.mdx

```diff
@@ -0,0 +1,619 @@
+---
+title: Distributed tracing for your PHP services
+tags:
+  - Agents
+  - PHP agent
+  - Features
+metaDescription: 'How to enable distributed tracing or Infinite Tracing for the APM PHP agent.'
+redirects:
+  - /docs/agents/php-agent/features/distributed-tracing-php-agent
+  - /docs/agents/php-agent/features/distributed-tracing
+  - /docs/agents/php-agent/features/distributed-tracing-php
+  - /docs/apm/agents/php-agent/features/distributed-tracing-php-agent
+---
+
+import phplogo from 'images/phplogo.png'
+
+import LogsInContexTraces from 'images/logs-in-context-traces.png'
+
+Distributed tracing allows you to see the entire journey of your requests throughout a [distributed system](/docs/distributed-tracing/concepts/introduction-distributed-tracing). The PHP agent automatically instruments with distributed tracing a number of native PHP functions, as well as some third party HTTP clients:
+
+* PHP function [`file_get_contents`](http://php.net/manual/en/function.file-get-contents.php)
+* PHP functions [`curl_exec`](http://php.net/manual/en/function.curl-exec.php) and [`curl_multi_exec`](http://php.net/manual/en/function.curl-multi-exec.php)
+* Guzzle 4, Guzzle 5, Guzzle 6
+* Drupal's `drupal_http_request function`
+* Laravel's queue jobs
+
+For the PHP agent, we offer two types of distributed tracing (for more, see [How span sampling works](/docs/understand-dependencies/distributed-tracing/get-started/how-new-relic-distributed-tracing-works#sampling)):
```

Comment:
@zuluecho9 I'm trying to avoid adding anything following the bullet list since I'm already juggling two bullet lists in the same section. How about this: (for more details, see...)?